innoweb/views.py

```diff
@@ -67,3 +71,31 @@ def home(request):
         'third': third
         })
 
+
+def certificate(request, event_id, participant_id):
+
+    e = Event.objects.get(id=event_id)
+    participant = Participant.objects.get(id=participant_id)
+
+    score = Score.objects.get(event=e, participant=participant)
+
+
+    certificate = Image.open("static\images\Sample_certificate.png")
+    certificate.load()
+    certificate = certificate.convert("RGB")
+
+
+    new = ImageDraw.Draw(certificate)
+
+    font = ImageFont.truetype(r"BRUSHSCI.TTF", 90)
+    font_small = ImageFont.truetype(r"BRUSHSCI.TTF", 50)
+    new.text((498, 518), participant.get_complete_name(), (0, 0, 0), font=font)
+    new.text((757, 620), e.name, (0, 0, 0), font=font_small)
+    new.text((560, 690), datetime.strftime(e.date, "%d/%m/%Y"), (0, 0, 0), font=font_small)
+    new.text((890, 690), e.place, (0, 0, 0), font=font_small)
+    new.text((1000, 757), str(score.value), (0, 0, 0), font=font_small)
+
+    certificate.save("static\certificates\Certificado.pdf", "PDF", resolution=180.0)
```

Comment:
This directory should be created, since when uploading the changes it does not exist and it gives an error when trying to generate a diploma.

Reply:
You are right. I've just changed it.